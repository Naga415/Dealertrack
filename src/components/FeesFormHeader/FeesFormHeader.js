"use strict";
import React from "react";
import Row from "@coxautokc/fusion-ui-components/lib/Row";
import Col from "@coxautokc/fusion-ui-components/lib/Col";
import "./FeesFormHeader.scss";

export default function FeesFormHeader() {
  return (
    <Row id="FeesFormHeader">
      <Col md={3} sm={3} id="feesTypeLabel">
        <label htmlFor="feesTypeLabel">
          Fee Type
        </label>
      </Col>
      <Col md={3} sm={3} id="feesNameLabel">
        <label htmlFor="feesNameLabel">
          Fee Name
        </label>
      </Col>
      <Col md={2} sm={2} id="feesAmountLabel">
        <label htmlFor="feesAmountLabel">
          Amount
        </label>
      </Col>
      <Col md={1} sm={1} id="feesCapLabel">
        <label htmlFor="feesCapLabel" className="fees__capLabel">
          Cap
        </label>
        <div className="fees__tooltip">
          <i className="fa fa-info-circle" />
          <span className="tooltiptext">Capitalize</span>
        </div>
      </Col>
      <Col md={2} sm={2} id="feesPaidToLabel">
        <label htmlFor="feesPaidToLabel">
          Paid To
        </label>
      </Col>
    </Row>
  );
}
