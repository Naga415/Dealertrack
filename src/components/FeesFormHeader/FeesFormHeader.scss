@import "~@coxautokc/fusion-theme/dist/fusion-variables.scss";

#FeesFormHeader {
  padding-top: 25px;

  label {
    font-weight: bold;
    min-height: 30px;
    color: #454545;
  }

  .fees__capLabel {
    margin-right: 4px;
  }

  .fees__tooltip {
    position: relative;
    display: inline-block;
  }

  .fees__tooltip .tooltiptext {
    visibility: hidden;
    background-color: #fff;
    color: #000;
    text-align: center;
    border: 1px solid #808080;
    padding: 5px 0;
    width: 120px;
    bottom: 100%;
    left: 50%;
    margin-left: -60px;
    opacity: 1 !important;

    /* Position the tooltip */
    position: absolute;
    transform-origin: 0 0;
    box-shadow: -3px 3px 3px 0 rgba(0, 0, 0, 0.4);
  }

  .tooltip-top {
    bottom: 125%;
    left: 50%;
    margin-left: -60px;
  }

  .fees__tooltip:hover .tooltiptext {
    visibility: visible;
  }

  .fees__tooltip .tooltiptext::after {
    content: " ";
    position: absolute;
    top: 100%; /* At the bottom of the tooltip */
    left: 48%;
    margin-left: -5px;
    border: 5px solid #fff;
    border-color: transparent transparent #fff #fff;
    transform-origin: 0 0;
    transform: rotate(-45deg);
    box-shadow: -3px 3px 3px 0 rgba(0, 0, 0, 0.4);
  }

  .col-lg-1,
  .col-lg-10,
  .col-lg-11,
  .col-lg-12,
  .col-lg-2,
  .col-lg-3,
  .col-lg-4,
  .col-lg-5,
  .col-lg-6,
  .col-lg-7,
  .col-lg-8,
  .col-lg-9,
  .col-md-1,
  .col-md-10,
  .col-md-11,
  .col-md-12,
  .col-md-2,
  .col-md-3,
  .col-md-4,
  .col-md-5,
  .col-md-6,
  .col-md-7,
  .col-md-8,
  .col-md-9,
  .col-sm-1,
  .col-sm-10,
  .col-sm-11,
  .col-sm-12,
  .col-sm-2,
  .col-sm-3,
  .col-sm-4,
  .col-sm-5,
  .col-sm-6,
  .col-sm-7,
  .col-sm-8,
  .col-sm-9,
  .col-xs-1,
  .col-xs-10,
  .col-xs-11,
  .col-xs-12,
  .col-xs-2,
  .col-xs-3,
  .col-xs-4,
  .col-xs-5,
  .col-xs-6,
  .col-xs-7,
  .col-xs-8,
  .col-xs-9 {
    padding-left: 5px;
    padding-right: 5px;
  }
}
