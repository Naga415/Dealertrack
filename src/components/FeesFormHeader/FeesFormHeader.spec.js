"use strict";
import React from 'react';
import renderer from 'react-test-renderer';
import FeesFormHeader from './FeesFormHeader';

describe('FeesFormHeader specs', () => {
  it('renders <FeesFormHeader />', () => {
    const component = renderer.create(<FeesFormHeader />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
