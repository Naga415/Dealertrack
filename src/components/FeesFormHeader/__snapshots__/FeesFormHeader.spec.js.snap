// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeesFormHeader specs renders <FeesFormHeader /> 1`] = `
<div
  className="row"
  id="FeesFormHeader"
>
  <div
    className="col-md-3 col-sm-3"
    id="feesTypeLabel"
  >
    <label
      htmlFor="feesTypeLabel"
    >
      Fee Type
    </label>
  </div>
  <div
    className="col-md-3 col-sm-3"
    id="feesNameLabel"
  >
    <label
      htmlFor="feesNameLabel"
    >
      Fee Name
    </label>
  </div>
  <div
    className="col-md-2 col-sm-2"
    id="feesAmountLabel"
  >
    <label
      htmlFor="feesAmountLabel"
    >
      Amount
    </label>
  </div>
  <div
    className="col-md-1 col-sm-1"
    id="feesCapLabel"
  >
    <label
      className="fees__capLabel"
      htmlFor="feesCapLabel"
    >
      Cap
    </label>
    <div
      className="fees__tooltip"
    >
      <i
        className="fa fa-info-circle"
      />
      <span
        className="tooltiptext"
      >
        Capitalize
      </span>
    </div>
  </div>
  <div
    className="col-md-2 col-sm-2"
    id="feesPaidToLabel"
  >
    <label
      htmlFor="feesPaidToLabel"
    >
      Paid To
    </label>
  </div>
</div>
`;
